common/src/chain/tokens/mod.rs

```diff
@@ -54,10 +56,15 @@ impl TokenAuxiliaryData {
 #[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Encode, Decode)]
 pub enum OutputValue {
     Coin(Amount),
-    Token(TokenData),
+    Token(Box<TokenData>),
 }
 
 impl OutputValue {
+    // FIXME(nft_issuance): Maybe we can use a better name
+    pub fn new_boxed_token(token_data: TokenData) -> Self {
```

Comment:
No need to call it boxed. It's an implementation detail.

Reply:
Cool